real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/Gui.java

```diff
@@ -23,7 +25,8 @@
 
 
 	//The windowList is a list of all of the windows that are a part of the gui
-	private ArrayList<RobobuggyJFrame> windowList = new ArrayList<RobobuggyJFrame>();
+    HashMap<Integer, RobobuggyJFrame> windowMap;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2324366005/issues/source?bid=2938665&fileBranchId=3205314#l28)